cortx-motr.spec.in

```diff
@@ -24,7 +25,11 @@
                         )
 
 %define kernel_ver %( echo %{raw_kernel_ver} | tr - _ | sed -e 's/\.debug$//' -e 's/\.%{_arch}$//' -e 's/\.%{?dist:-el7}$//' -e 's/\.el7$//'  )
+%if  "%{?CONFIG_ARCH_TYPE}" == "x86_64"
```

Comment:
Double-space after `if`. Fix it here and everywhere in your new code, please.

Reply:
Fixed